src/ui/components/DevTools.tsx

```diff
@@ -225,7 +225,13 @@ function _DevTools({
     });
   });
 
+  const didInitializeSocketRef = useRef<boolean>(false);
   useEffect(() => {
+    if (didInitializeSocketRef.current) {
+      return;
+    }
+    didInitializeSocketRef.current = true;
+
     async function createSocketWithToken() {
```

Comment:
I'm not completely sure if this is never meant to get called again (like with a new `accessToken`). Based on a quick reading of the code that would always~ trigger the mentioned error so I assume it shouldn't be called more than once

Reply:
+ from what I can tell `setAccessToken` could also be removed, I can't find any dispatch site for it